app/components/FilterableTable/Filters/TableFilter.tsx

```diff
@@ -56,6 +55,7 @@ export default abstract class TableFilter<T = string | number | boolean> {
           name={this.argName}
           value={(filterArgs[this.argName] ?? '') as string | number}
           aria-label={`Filter by ${this.title}`}
+          disabled={!this.isSearchEnabled}
```

Comment:
I guess this is a bit of a new feature but this should prevent confusion if the search is disabled by the dev